actions/namespace-create

```diff
@@ -15,7 +15,8 @@ os.environ['PATH'] += os.pathsep + os.path.join(os.sep, 'snap', 'bin')
 
 
 def kubectl(args):
-    cmd = ['kubectl'] + args
+    cmd = ["kubectl", "--kubeconfig=/home/ubuntu/config"]
```

Comment:
Question: would it be better to use `$HOME` instead of `/home/ubuntu`?

Reply:
> Question: would it be better to use `$HOME` instead of `/home/ubuntu`?

Pon, $HOME is not defined for execution of actions and hence the problem. Please see [1]
I have used /home/ubuntu instead of /root/.kube just to make this consistent across other actions
  
[1] https://bugs.launchpad.net/charm-kubernetes-master/+bug/1914359/comments/1

Reply:
Thanks, Hemanth!